### PR TITLE
SPLICE-1970 Exclude metrics jars from splice-uber jar to avoid class loader issues when using Spark Adapter(2.5)

### PR DESCRIPTION
**Pull request by stpabhi**

### CODE REVIEW

File: assembly/pom.xml [hunk 1]
```diff
@@ -94,6 +94,11 @@
                                     <exclude>com.splicemachine:db-tools-ij</exclude>
                                     <exclude>com.splicemachine:db-shared</exclude>
                                     <exclude>com.splicemachine:db-drda</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-core</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-graphite</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-json</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-jvm</exclude>
+                                    <exclude>com.yammer.metrics:metrics-core</exclude>
```
Comment (OlegMazurov) [inline assembly/pom.xml]: Please use io.dropwizard.metrics:* and com.yammer.metrics:*

File: assembly/pom.xml [hunk 2]
```diff
@@ -237,10 +242,13 @@
                                 db-shared,
                                 db-drda,
                                 javax.servlet-api,
-                                javax.ws.rs-api
+                                javax.ws.rs-api,
+                                metrics-core,
+                                metrics-graphite,
+                                metrics-json,
+                                metrics-jvm
```
Comment (OlegMazurov) [inline assembly/pom.xml]: Why are these needed?

File: assembly/pom.xml (hunk 1 above)
Comment (stpabhi) [reply assembly/pom.xml]: Ok

File: assembly/pom.xml (hunk 2 above)
Comment (stpabhi) [reply assembly/pom.xml]: Good question. These are not needed unless splice has any dependency on these versions. I wasn’t sure about it so added them in lib.

Comment (OlegMazurov) [reply assembly/pom.xml]: All dependencies on metrics are transitive only.

Comment (stpabhi) [reply assembly/pom.xml]: Ok, will remove them

**Comment by stpabhi** [comment]
@OlegMazurov I have made the suggested changes.
